### PR TITLE
Update advanced-permissions.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Reorganized order of UI steps, fixed some erroneous punctuation, general editing.

### CODE REVIEW

File: source/deployment/advanced-permissions.rst [hunk 1]
```diff
@@ -76,21 +76,27 @@ Allows Admins to grant additional permissions to specific users or to a group of
 
 Recipes
 --------
-This section provides some examples of common permissions use cases and how to accomplish them using the Advanced Permissions System Console interface.
+This section provides some examples of common permissions use cases and how to accomplish them using the **Advanced Permissions System Console interface.
```
Comment (amyblais) [inline source/deployment/advanced-permissions.rst]:
```suggestion
This section provides some examples of common permissions use cases and how to accomplish them using the **Advanced Permissions System Console interface**.
```

File: source/deployment/advanced-permissions.rst [hunk 2]
```diff
@@ -76,21 +76,27 @@ Allows Admins to grant additional permissions to specific users or to a group of
...
 
 Team Management
 ~~~~~~~~~~~~~~~~
 
-**Restrict adding members to only Admins, in specific teams**
+**Only allow Admins, in specific team, to add memberss**
```
Comment (amyblais) [inline source/deployment/advanced-permissions.rst]:
```suggestion
**Only allow Admins, in specific team, to add members**
```

File: source/deployment/advanced-permissions.rst [hunk 3]
```diff
@@ -76,21 +76,27 @@ Allows Admins to grant additional permissions to specific users or to a group of
...
 Recipes
 --------
-This section provides some examples of common permissions use cases and how to accomplish them using the Advanced Permissions System Console interface.
+This section provides some examples of common permissions use cases and how to accomplish them using the **Advanced Permissions System Console interface**.
 
 Team Management
 ~~~~~~~~~~~~~~~~
 
-**Restrict adding members to only Admins, in specific teams**
+**Only allow Admins, in specific team, to add members**
```
Comment (amyblais) [inline source/deployment/advanced-permissions.rst]:
```suggestion
**Only allow Admins, in specific team, to add members**

```

File: source/deployment/advanced-permissions.rst [hunk 4]
```diff
@@ -76,21 +76,27 @@ Allows Admins to grant additional permissions to specific users or to a group of
...
-2. In the **All Members** panel, check the box for **Add Team Members**. This sets the system default for all teams.
-3. In **System Console** > **Advanced Permissions** in prior versions or **System Console** > **User Management** > **Permissions** in versions after 5.12, create a new **Team Override Scheme**.
-  i. Name and describe the scheme. For example, ``Authorized Personel Only`` with description ``Restrict adding team members to Team and System Admins.``
+1. Navigate to **System Console** > **User Management** > **Permissions** (or **System Console** > **Advanced Permissions** in versions prior to 5.12)
+2. Select **Edit Scheme**.
+3. In the **All Members > Teams** panel, check the box for **Add Team Members**. This sets the system default for all teams.
+4. Choose **Save**. 
+5. Select the back arrow to return to the **Permission Schemes** menu. 
+
+1. Seect **New Team Override Scheme**.
```
Comment (amyblais) [inline source/deployment/advanced-permissions.rst]:
```suggestion
1. Select **New Team Override Scheme**.
```

Comment (amyblais) [inline source/deployment/advanced-permissions.rst]: Should these be step 6-8?

File: source/deployment/advanced-permissions.rst [hunk 5]
```diff
@@ -99,44 +105,45 @@ Public and Private Channel Management
...
+2. Select **Edit Scheme**.
+3. In the **All Members** panel, uncheck the box for **Manage Public Channels > Manage Channel Settings**.
+4. In the **Channel Administrator** and **Team Administrator** panels, check the box for **Manage Public Channels > Manage Channel Settings**.
 
 .. note::
 
-  Permissions for channel renaming, editing header and editing purpose are currently grouped in a single permission. These will be split into separate permissions in a future release.
+  Permissions for channel renaming, editing header, and editing purpose are currently grouped in a single permission. These will be split into separate permissions in a future release.
 
 **Restrict who can create channels, in specific teams**
```
Comment (amyblais) [inline source/deployment/advanced-permissions.rst]: Add space

File: source/deployment/advanced-permissions.rst (hunk 4 above)
Comment (justinegeffen) [reply source/deployment/advanced-permissions.rst]: Good question - it's the start of a new series of steps, which is why I numbered it from 1 again. If it makes more sense to start at 6 until 8 I can do that.

File: source/deployment/advanced-permissions.rst [hunk 6]
```diff
@@ -99,44 +105,45 @@ Public and Private Channel Management
...
   i. Name and describe the scheme. For example, ``Contractor Scheme`` with description ``Restrict public channel creation to Admins only``.
   ii. Add Team C to the **Select teams to override permissions** list.
   iii. In the **All Members** panel, uncheck the box for **Create Channels** in the **Manage Public Channels** section.
-  iv. In the **Team Admin** and **Channel Admin** panels, check the box for **Create Channels** in the **Manage Public Channels** section.
+  iv. In the **Channel Administrator** and **Team Administrator** panels, check the box for **Create Channels** in the **Manage Public Channels** section.
 
 Post Management
 ~~~~~~~~~~~~~~~~
 
 **Restrict who can delete posts**
```
Comment (amyblais) [inline source/deployment/advanced-permissions.rst]: Add space

File: source/deployment/advanced-permissions.rst [hunk 7]
```diff
@@ -99,44 +105,45 @@ Public and Private Channel Management
...
 
-1. In **System Console** > **Advanced Permissions** in prior versions or **System Console** > **User Management** > **Permissions** in versions after 5.12, edit the **System Scheme**.
-2. In the **All Members** and **Channel Admin** panels, uncheck the boxes for **Delete Own Posts** and **Delete Others Posts**.
-3. In the **Team Admin** and **Channel Admin** panels, check the box boxes for **Delete Own Posts** and **Delete Others Posts**.
+1. Navigate to **System Console** > **User Management** > **Permissions** (or **System Console** > **Advanced Permissions** in versions prior to 5.12).
+2. Select **Edit Scheme**.
+3. In the **All Members** and **Channel Admin** panels, uncheck the boxes for **Delete Own Posts** and **Delete Others Posts**.
+4. In the **Channel Administrator** and **Team Administrator** panels, check the box boxes for **Delete Own Posts** and **Delete Others Posts**.
 
 **Restrict who can edit posts**
```
Comment (amyblais) [inline source/deployment/advanced-permissions.rst]: Add space

File: source/deployment/advanced-permissions.rst [hunk 8]
```diff
@@ -174,11 +186,11 @@ Technical Admins or developers looking for a deeper understanding of the permiss
...
-- Group: A set of users, usually synced from AD/LDAP. Groups are assigned to roles in the context of teams, channels or system-wide.
-- Default Roles: System Admin, Team Admin, Channel Admin, Member.
-- System Scheme: A set of default roles that apply system wide
-- Team Override Scheme: A set of default roles that apply only in the team specified. Permissions granted to roles in a team scheme override roles in the system scheme.
-- System-wide: Applies across the entire system, including all teams of which the user is a member.
-- Team Wide: Applies in a specific team only.
+- **Permission:** The ability to execute certain actions. Permissions are granted to roles.
+- **Roles:** A set of permissions. Users or groups are assigned to roles.
+- **Group:** A set of users, usually synced from AD/LDAP. Groups are assigned to roles in the context of teams, channels, or system-wide.
+- **Default Roles:** All Members, Channel Administrators, Team Administrators, System Administrators.
```
Comment (wiersgallak) [inline source/deployment/advanced-permissions.rst]: Wondering if we want to add Guest Accounts here (if it is enabled).

**Comment by justinegeffen** [comment]
Thanks @wiersgallak. I've marked this as under review as I noticed there are some erroneous steps.

File: source/deployment/advanced-permissions.rst [hunk 9]
```diff
@@ -16,43 +16,44 @@ Advanced permissions offers Admins a way to restrict actions in Mattermost to au
...
 
-- Team Admins: permissions granted apply to all Team Admins, in all teams.
-- Channel Admins: permissions granted apply to all Channel Admins in all channels, in all teams.
-- All Members: permissions granted apply to all members, including Admins, in all channels, in all teams. 
-- Guests: permissions granted to users with the Guest role in all channels and teams. 
+- **Guests:** If Guest Accounts are enabled, permissions apply to guest users in all channels, in all teams.
+- **All Members:** Permissions apply to all members, including Admins, in all channels, in all teams. 
+- **Channel Administrators:** Permissions apply to all Channel Admins in all channels, in all teams.
+- **Team Administrators:** Permissions apply to all Team Admins, in all teams.
+- **Guests:** Permissions granted to users with the Guest role, in all channels and teams.
```
Comment (amyblais) [inline source/deployment/advanced-permissions.rst]: Guests is doubled in this list.

File: source/deployment/advanced-permissions.rst (hunk 4 above)
Comment (amyblais) [reply source/deployment/advanced-permissions.rst]: One of these options works:

1. There should be a title+description in between so it's clear why there are two lists.
2. The second list can be intended similar to how it's [done here](https://github.com/mattermost/docs/blob/8b6080708775f6fe3ed8770ea77e1ed48b36f835/source/deployment/advanced-permissions.rst#team-management).

Comment (justinegeffen) [reply source/deployment/advanced-permissions.rst]: I chose option two, thanks for the suggestions - I didn't think of it like that. :)

**Pull request merged**